vizro-core/docs/pages/user-guides/run.md

```diff
@@ -4,13 +4,14 @@ This guide shows you how to launch your dashboard in different ways. By default,
 
 ## Py.Cafe
 
-You can launch your dashboard and edit the code live on [Py.Cafe](https://py.cafe/).
+The easiest way to launch your dashboard is to edit the code live on [Py.Cafe](https://py.cafe/).
 
-For this basic example, follow the [Edit live on Py.Cafe](https://py.cafe/stichbury/vizro-iris-analysis) link below the dashboard to experiment with the code.
+Most of our examples in the docs have link below the code - [Edit live on Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) - which you can follow to experiment with the code of that example. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.
```

Comment:
```suggestion
Most of our examples have a link below the code — [Edit live on Py.Cafe](https://py.cafe/vizro./vizro-iris-analysis-0) — which you can follow to experiment with the code. This will lead you to an editor such as the one below, which displays the dashboard and the code side by side.
```